example/state-codes.jv

```diff
@@ -0,0 +1,78 @@
+// SPDX-FileCopyrightText: 2023 Friedrich-Alexander-Universitat Erlangen-Nurnberg
+//
+// SPDX-License-Identifier: AGPL-3.0-only
+
+// This file belongs to example 2: Electric Vehicles
+// Learning goals:
+// - Understand how to publish elements to use them in other files
+
+// 1. The publish keyword can be used directly when defining an element.
+publish valuetype UsStateCode oftype text {
```

Comment:
US is an abbreviation, not a word 😅 . Changing this means changing it in all comments / references as well sadly.

```suggestion
publish valuetype USStateCode oftype text {
```